cluster_preflight_check/main.py

```diff
@@ -371,6 +371,10 @@ def run(context):
     '''
     major work flow
     '''
+    if os.getuid() != 0:
+        print("Error: {} can only be executed as user root!".format(context.name))
```

Comment:
I would suggest to output any error message to stderr (maybe you need to add a `import sys` line somewhere):

```suggestion
        print("Error: {} can only be executed as user root!".format(context.name), file=sys.stderr)
```

As an alternative, you could use the logger module and output it with `logger.fatal` or `logger.critical`. Would it make sense to move the if clause *after* line 378? I have the following in my mind:

```python
setup_logging(context)
if os.getuid() != 0:
    logger.fatal("Error: %s can only be executed as user root!", context.name)
    sys.exit(1)
# ...
```